Products/ZenHub/tests/test_zenhub.py

```diff
@@ -1235,3 +1236,28 @@ def test_check(t):
         zenhub = sentinel.zenhub
         default_hub_heartbeat_check = DefaultHubHeartBeatCheck(zenhub)
         default_hub_heartbeat_check.check()
+
+
+class TestModelingPaused(TestCase):
+
+    def test_paused(self):
+        dmd = Mock()
+        dmd.getPauseADMLife.return_value = 100
+        pause_timeout = 200
+        paused = ModelingPaused(dmd, pause_timeout)
+
+        actual = paused()
+
+        dmd.getPauseADMLife.assert_has_calls([call()])
```

Comment:
more concise way to say 'called with no parameters'
```suggestion
        dmd.getPauseADMLife.assert_called_with()
```

Reply:
Hmm... I was hoping to simulate `assert_called_once` (not available in the version of mock we're using), but neither `assert_has_calls([call()])` nor `assert_called_with()` appear to do that.